bitbots_blackboard/src/bitbots_blackboard/capsules/team_data_capsule.py

```diff
@@ -26,7 +29,7 @@ def get_team_goalie_ball_position(self):
         """
         i = 0
         for state in self.team_data.state:
-            if state is TeamData.ROLE_GOALIE:
+            if state is TeamData.ROLE_GOALIE: #TODO: should this be Strategy.ROLE_GOALIE?
```

Comment:
```suggestion
            if state == Strategy.ROLE_GOALIE:
```
`should this be Strategy.ROLE_GOALIE?` yes, there is no `TeamData.ROLE_GOALIE`.